src/node/build.ts

```diff
@@ -33,7 +33,7 @@ export async function build(ssgOptions: Partial<ViteSSGOptions> = {}, viteConfig
 
   const cwd = process.cwd()
   const root = config.root || cwd
-  const ssgOut = join(root, '.vite-ssg-temp')
+  const ssgOut = join(root, '.vite-ssg-temp-' + Math.random().toString(36).substring(2, 12))
```

Comment:
```suggestion
  const ssgOut = join(root, '.vite-ssg-temp', Math.random().toString(36).substring(2, 12))
```

Maybe using subfolder would help gitignore